desktop-widgets/downloadfromdivecomputer.cpp

```diff
@@ -117,10 +117,15 @@ DownloadFromDCWidget::DownloadFromDCWidget(const QString &filename, QWidget *par
 #define SETUPDC(num) \
 	if (!qPrefDiveComputer::vendor##num().isEmpty()) { \
 		ui.DC##num->setVisible(true); \
+		ui.DCFrame##num->setVisible(true); \
```

Comment:
This has become a bit messy now, and includes changes from a number of past pull requests that have  been merged already.
The source of the messiness is that you are re-using the `master` branch in your fork for all of your pull requests - this is a recipe for problems.
Try instead creating a new branch in your fork for each new pull request, make the changes in this branch, and then open a pull request from this branch into `master` in the `subsurface/subsurface` repository.